argon2/src/params.rs

```diff
@@ -233,6 +236,14 @@ macro_rules! param_buf {
                 Ok(Self { bytes, len })
             }
 
+            /// Just like [Default::default], but const.
+            pub const fn default_const() -> Self {
+                Self {
+                    bytes: [0u8; Self::MAX_LEN],
+                    len: 0,
+                }
+            }
```

Comment:
This could instead be:

```suggestion
            /// Empty value.
            pub const EMPTY: Self = Self {
                bytes: [0u8; Self::MAX_LEN],
                len: 0,
            };
```

Reply:
Also this could be used in the definition of `Params::DEFAULT`